tests/validate_typecheck_testfile.testscript

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Nit: as best practice, I tend to prefer `#!/usr/bin/env bash` to `#!/bin/bash`.

(I realise this convention isn't followed throughout the rest of the codebase, and it really is nit-picking)

Reply:
Good point. 

Might be worth mentioning why, namely, that writing `/usr/bin/env bash` in a shebang is portable whereas `/bin/bash` is not. However as we do not officially support any other platforms than Linux, the latter ought to work

Reply:
The latter will be fine. I think it's just habit that was beaten into me at St Andrews.

Reply:
Given that it's a tiny change we should just insist on doing the portable thing everywhere.